File: Modul 5/PRAK502_2410817310016_ARIFFATAHILLAHPERMANA.py
def hitung(nilai1, nilai2):
    return nilai1 - nilai2 

def mutlak(angka):
    return -angka if angka < 0 else angka 

def main():
    a, b, c, d = map(int, input().split())
    Hasil = mutlak(hitung(a, c)) + mutlak(hitung(b, d))
    print(Hasil)

if __name__ == "__main__":
    main()

File: Modul 5/PRAK502_2410817310016_ARIFFATAHILLAHPERMANA.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2) {
    return nilai1 - nilai2; 
}

int mutlak(int angka) {
    return (angka < 0) ? -angka : angka;
}

int main() {
    int a, b, c, d;
    scanf("%d", &a); 
    scanf("%d", &b); 
    scanf("%d", &c); 
    scanf("%d", &d); 
    int Hasil = mutlak(hitung(a, c)) + mutlak(hitung(b, d)); 
    printf("%d", Hasil); 
    return 0;
}